File: Java_HDOJ/src/HDOJ2009.java
import java.util.Scanner;

public class HDOJ2009 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int i,m;
        double n,t,result;
        while(sc.hasNext()){
            n= sc.nextDouble();
            m= sc.nextInt();
            result=n;
            t=n;
            for(i=0;i<m-1;i++){
                n=Math.sqrt(n);
                result+=n;
            }
            System.out.printf("%.2f",result);
            System.out.println();
        }
    }
}

File: C_HDOJ/HDOJ2009.c
#include <stdio.h>
#include <math.h>

int main(){
	int i,m;
	float n,t,result;
	while(~scanf("%f %d",&n,&m)){
		result=n;
		t=n;
		for(i=0;i<m-1;i++){
			n=sqrt(n);
			result+=n;
		}
		printf("%.2f\n",result);
	}
	return 0;
}